oauthproxy.go

```diff
@@ -858,9 +858,17 @@ func (p *OAuthProxy) Proxy(rw http.ResponseWriter, req *http.Request) {
 	session, err := p.getAuthenticatedSession(rw, req)
 	switch err {
 	case nil:
+		//<!--COCO Begin-->
+		// COCO: check session token present in cookie or not
+		cookie, err := req.Cookie("sessionToken")
+		if err != nil {
+			fmt.Printf("can't find sessionToken redireacting to logout page")
+			p.SignOut(rw, req)
+		}
 		// we are authenticated
 		p.addHeadersForProxying(rw, req, session)
 		p.serveMux.ServeHTTP(rw, req)
+		//<!--COCO End-->
```

Comment:
- this should be after the above if condition. i.e., after line # 867

Reply:
handled